server/mscalendar/availability.go

```diff
@@ -199,7 +199,7 @@ func (m *mscalendar) setStatusFromCalendarView(user *store.User, currentStatus s
 			}
 			message = "User is no longer busy in calendar. Set status to online."
 		} else {
-			message = "User is no longer busy in calendar, but is not set to busy. No status change."
+			message = fmt.Sprintf("User is no longer busy in calendar, but is not set to busy(%s). No status change.", busyStatus)
```

Comment:
Non-blocking nit, I'm thinking there should be a space between "busy" and the busyStatus

```suggestion
			message = fmt.Sprintf("User is no longer busy in calendar, but is not set to busy (%s). No status change.", busyStatus)
```